sdl-core/src/main/scala/io/smartdatalake/workflow/dataobject/Table.scala

```diff
@@ -30,7 +30,23 @@ import org.apache.spark.sql.catalyst.TableIdentifier
  *           Called db for backwards-compatibility because for hive tables, db and schema mean the same thing.
  * @param name        table name
  * @param query       optional select query
+ * @param commentOnTable  An optional comment to add to the table after writing a DataFrame to it. As of now, this is only possible for Delta Lake and Snowflake tables.
+ * @param commentsOnColumns Optional sequence of comments to add to the columns of the table as metadata. They must be written as follows:
```

Comment:
Also not sure that we should introduce another attribute to define column comments. See also my last comment https://github.com/smart-data-lake/smart-data-lake/issues/865.